crowbar_framework/app/models/nova_service.rb

```diff
@@ -188,10 +188,11 @@ def apply_role_pre_chef_call(old_role, role, all_nodes)
       # TODO(toabctl): The same code is in the neutron barclamp. Should be extracted and reused!
       #                (see crowbar_framework/app/models/neutron_service.rb)
       if neutron["attributes"]["neutron"]["networking_plugin"] == "ml2"
-        if neutron["attributes"]["neutron"]["ml2_type_drivers"].include?("gre")
+        ml2_type_drivers = neutron["attributes"]["neutron"]["ml2_type_drivers"]
+        if ml2_type_drivers.include?("gre") || ml2_type_drivers.include?("vxlan")
```

Comment:
I dislike this, the current code is better readable especially as many people don't know what the ```&``` operator does in this case.